qemu/tests/cfg/balloon_small_size.cfg

```diff
@@ -0,0 +1,12 @@
+- balloon_small_size:
+    virt_test_type = qemu
+    no Win2000
+    only Windows
```

Comment:
According to our discuss, you case of "small size" belongs to test case "boundary", not this category "small_size". Got it?